yarn-error.log

```diff
@@ -0,0 +1,92 @@
+Arguments: 
```

Comment:
You probably don't want to track this under version control?

Reply:
true, I'll remove this